app/views/foreman/job_templates/flatpak_setup.erb

```diff
@@ -0,0 +1,37 @@
+<%#
+kind: job_template
+name: Install Flatpak and Firefox
+job_category: Katello
+description_format: 'Install Flatpak and Firefox'
+feature: katello_host_flatpak_install
+provider_type: script
+template_inputs:
+- name: Flatpak registry URL
+  description: URL of server/capsule
+  input_type: user
+  required: true
+- name: Username
+  description: Username for container registry login
+  input_type: user
+  required: true
+- name: Password
+  description: Password for container registry login
```

Comment:
This is where we're meant to use the personal auth token, right? Maybe the input or description should call that out.